cmd/apps/helm_operator_app.go

```diff
@@ -0,0 +1,171 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallHelmOperator() *cobra.Command {
+	var operator = &cobra.Command{
+		Use:          "helm-operator",
+		Short:        "Install a the helm operator",
```

Comment:
"Install a the helm operator" <- "Install the helm operator"

Reply:
Done